src/controller/avdeccControllerImplHandlers.cpp

```diff
@@ -98,20 +98,6 @@ void ControllerImpl::onConfigurationDescriptorResult(entity::ControllerEntity co
 					// Only get full descriptors for active configuration
 					if (isCurrentConfiguration)
 					{
-						// Get Locales
```

Comment:
This one was there by choice actually (was planning on exposing the entity even if it's not completely enumerated.... some day, so the localization should be accessible first).
This code is not tight to the protocol itself, it's the enumeration state machine of the controller. Almost nothing is defined in IEEE1722.1 on how (nor in which order) a controller should get the information from an entity.

Everything else is good, ty for updating

Reply:
Oh sorry for that - I have reverted it.